hatchet_sdk/contracts/dispatcher_pb2.pyi

```diff
@@ -1,5 +1,3 @@
-# type: ignore
```

Comment:
We want to remove this? Thought that's what was making mypy work

Reply:
ah we had auto merge on... let's fix mypy in separate PR